wsimod/nodes/land.py

```diff
@@ -351,6 +376,44 @@ def __init__(
         self.processes = []
         self.outflows = []
 
+    def apply_overrides(self, overrides = Dict[str, Any]):
+        """Override parameters.
+
+        Enables a user to override any of the following parameters: 
+        area and depth (both will update the capacity), pollutant_load (the
+        entire dict does not need to be redefined, only changed values need to
+        be included).
+
+        Args:
+            overrides (Dict[str, Any]): Dict describing which parameters should
+                be overridden (keys) and new values (values). Defaults to {}.
+        """
+        self.surface = overrides.pop("surface", 
+                                  self.surface)

```

Comment:
Is this something that someone would be doing? I wonder if it would interact with data reading?

Reply:
Depends on our choices of promoting the 'standard' way I guess, but happy to close this portal if we only want the user to change the surface name via yaml.

Reply:
I think remove it probably - otherwise we will need to figure out whether a bunch of stuff (i.e., things around the `data_input_dict` reading/loading/etc) would need to change. We can make a separate (low priority) issue for this if you think.